### PR TITLE
version: fix failing test

### PR DESCRIPTION
**Pull request by anhldbk**
## Issue reference

https://github.com/dapr/cli/issues/823

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Comment by anhldbk** [comment]
@artursouza @yaron2  PTAL. The failed test does NOT regard the change.

File: pkg/version/version_test.go [hunk 1]
```diff
@@ -114,7 +115,8 @@ func TestGetVersionsGithub(t *testing.T) {
 	t.Run("error on bad addr", func(t *testing.T) {
 		version, err := GetLatestReleaseGithub("http://a.super.non.existant.domain/")
 		assert.Equal(t, "", version)
-		assert.EqualError(t, err, "Get \"http://a.super.non.existant.domain/\": dial tcp: lookup a.super.non.existant.domain: no such host")
+		msg := fmt.Sprint(err)
+		assert.True(t, strings.Contains(msg, "no such host") || strings.Contains(msg, "i/o timeout"))
```
Comment (wcs1only) [inline pkg/version/version_test.go:119]: It is probably fine to just assert.Error(). The content of the error isn't so much important, and isn't necessarily directly under our control.

Comment (anhldbk) [reply pkg/version/version_test.go]: @wcs1only agree. I fixed it.

**Comment by anhldbk** [comment]
@wcs1only PTAL. 

@mukundansundar The failing test is related to e2e upgrade. https://github.com/dapr/cli/runs/4173343459?check_suite_focus=true. 

I think we have bugs in such e2e tests. I encountered such bug in another PR: https://github.com/dapr/cli/pull/770

**Review by mukundansundar — APPROVED** [review]

**Review by wcs1only — APPROVED** [review]

**Comment by anhldbk** [comment]
@wcs1only will this PR be merged?

**Pull request merged**